src/piper_cmd2_name_lexer.xrl

```diff
@@ -0,0 +1,20 @@
+Definitions.
+
+NAME                        = [a-z][a-zA-Z0-9_\-]*
+HIPCHAT_EMOJI               = \([a-zA-Z0-9_\-\+]+\)
+SLACK_EMOJI                 = :[a-zA-Z0-9_\-\+]+:
```

Comment:
Are we still wanting to support emoji in commands? While it's not a huge deal, it is rather odd seeing chat-adapter specific code down in the command parser.